src/main/java/brave/opentracing/BraveSpanBuilder.java

```diff
@@ -81,61 +71,47 @@
     return this;
   }
 
-  /**
-   * {@inheritDoc}
-   */
   @Override public Tracer.SpanBuilder withTag(String key, String value) {
     tags.put(key, value);
     return this;
   }
 
-  /**
-   * {@inheritDoc}
-   */
   @Override public Tracer.SpanBuilder withTag(String key, boolean value) {
     return withTag(key, Boolean.toString(value));
   }
 
-  /**
-   * {@inheritDoc}
-   */
   @Override public Tracer.SpanBuilder withTag(String key, Number value) {
     return withTag(key, value.toString());
   }
 
-  /**
-   * {@inheritDoc}
-   */
   @Override public Tracer.SpanBuilder withStartTimestamp(long microseconds) {
     this.timestamp = microseconds;
     return this;
   }
 
-  @Override
-  public Span start() {
+  @Override public Span start() {
     return startManual();
   }
 
-  @Override
-  public ActiveSpan startActive() {
+  @Override public Scope startActive() {
+    return startActive(true);
```

Comment:
I was wondering about this.. the default behavior is to finish the span when it goes out of scope? Seems opposite unless we expect most instrumentation to wrap synchronous calls that are enclosed by a single code block or method.

Reply:
Yes, the current scope impl closes it automatically.

I also think that a better sensible default value is *do not finish automatically*. Auto-finish makes sense for end users which rarely write tracing code.

Reply:
what happened to this, by the way? I saw in another issue opentracing people are going to 1.0 soon. Did this policy get reverted or is it locked in (scope stopping the span on close)